src/utils/testId.ts

```diff
@@ -4,4 +4,9 @@ export default {
     registerButton: 'login-register-button',
     showPasswordButton: 'login-show-password-icon-button',
   },
+  typeRegister: {
+    login: 'make-your-login',
+    iAmProfessor: 'register-professor',
+    iAmStudent: 'register-student',
```

Comment:
Vamo padronizar colocando o tipo de componente no final e o nome da tela no inicio?
```suggestion
    loginButton: 'register-login-button',
    iAmProfessorButton: 'register-professor-button',
    iAmStudentButton: 'register-student-button',
```

Reply:
Boa, não sabia se tinha um padrão mesmo. Feito